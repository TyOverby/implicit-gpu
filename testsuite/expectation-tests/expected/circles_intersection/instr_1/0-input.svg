version https://git-lfs.github.com/spec/v1
oid sha256:bacc5a6efe703fb35fa34563828e46d315fc4f3bc937158ae26b33f1e2600b51
size 6589
