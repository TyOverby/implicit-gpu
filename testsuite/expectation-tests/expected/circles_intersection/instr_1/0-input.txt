((11.0,21.0), (11.0,21.0))
((11.0,21.0), (11.0,21.0))
((11.050152,20.0), (11.202392,19.0))
((11.202392,19.0), (11.461792,18.0))
((11.461792,18.0), (11.835978,17.0))
((11.835978,17.0), (12.0,16.661701))
((12.0,16.661701), (12.343098,16.0))
((12.0,20.949848), (11.050152,20.0))
((12.343098,16.0), (13.0,15.0))
((13.0,15.0), (13.0,15.0))
((13.0,15.0), (13.862445,14.0))
((13.0,20.79761), (12.0,20.949848))
((13.862445,14.0), (14.0,13.862445))
((14.0,13.862445), (15.0,13.0))
((14.0,20.538208), (13.0,20.79761))
((15.0,13.0), (15.0,13.0))
((15.0,13.0), (16.0,12.343098))
((15.0,20.164022), (14.0,20.538208))
((15.338299,20.0), (15.0,20.164022))
((16.0,12.343098), (16.661701,12.0))
((16.0,19.656902), (15.338299,20.0))
((16.661701,12.0), (17.0,11.835978))
((17.0,11.835978), (18.0,11.461792))
((17.0,19.0), (16.0,19.656902))
((17.0,19.0), (17.0,19.0))
((18.0,11.461792), (19.0,11.202392))
((18.0,18.137556), (17.0,19.0))
((18.137556,18.0), (18.0,18.137556))
((19.0,11.202392), (20.0,11.050152))
((19.0,17.0), (18.137556,18.0))
((19.0,17.0), (19.0,17.0))
((19.656902,16.0), (19.0,17.0))
((20.0,11.050152), (20.949848,12.0))
((20.0,15.338299), (19.656902,16.0))
((20.164022,15.0), (20.0,15.338299))
((20.538208,14.0), (20.164022,15.0))
((20.79761,13.0), (20.538208,14.0))
((20.949848,12.0), (20.79761,13.0))
((21.0,11.0), (21.0,11.0))
((21.0,11.0), (21.0,11.0))
